### ISSUE TITLE
Update extension validation rules for acts_o_population_of

### ISSUE THREAD

**Issue by dustine32** [geneontology/gocamgen#75]
From https://github.com/geneontology/gocamgen/issues/63#issuecomment-561676086:

- [ ] 'stem cell population maintenance' (GO:0019827) should allow acts_on_population_of
- [ ] 'regulation of stem cell population maintenance' (GO:2000036) should allow regulates_o_acts_on_population_of

**Comment by dustine32** [comment]
@ukemi Yep! I added those two items to the rules, reran the report on the 2019-12-03 MGI GPAD, and the only one(!) line remaining was that missed line for 'negative regulation of oxidoreductase activity' regulates(GO:F).

Out of formality, here's the report:
https://docs.google.com/spreadsheets/d/1Z-D_qAcNpIUB-tMNiUbaQBuPRrbxVJoN8e7e1_ZzFh8/edit?folder=1sQAeWJDEk-joMCIvuoUt5A1ERahbjlv4#gid=0

**Comment by ukemi** [comment]
Sweet!

**Comment by ukemi** [comment]
OK @dustine32 and @vanaukenk, I have reread the old paper and changed the annotation to something that I think is a little bit better and conforms to the specs. Six days later than my GANTT chart indicated, but the MGI file should be clean with the next release. Thanks for all the help with this!!
